axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+they should be interpreted.
+
+It's important to note that the availability of these spans is highly dependent on the application configuration. 
+For instance, some components are only used when using Axon Server, or you might have created your own `CommandBus` 
+implementation which does not call the `SpanFactory` API. 
+
+### Commands
+
+The `CommandBus` is instrumented to create spans for both dispatching and handling commands. 
+
+When using the `AxonServerCommandBus`, there will be two handling and dispatch traces 
+since it uses a second `CommandBus` to invoke the command locally after receiving it from Axon Server. In addition,
+you can see the GRPC-call to Axon Server and the time it took to handle the call.
+
+The traces will, in sequence, look like this:
+
+| Trace name                                      | Description                                                                                                                                                                    |
+|-------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `AxonServerCommandBus.dispatch(${commandName})` | (Only with Axon Server) The bus is dispatching the command to Axon Server.                                                                                                     |
+| `AxonServerCommandBus.handle(${commandName})`   | (Only with Axon Server) The bus has received a command and is handling it.                                                                                                     |
+| `${CommandBusClass}.dispatch(${commandName})`   | The bus is dispatching the command locally (If using Axon Server: this is invoked by the `AxonServerCommandBus`.                                                               |
+| `${CommandBusClass}.handle(${commandName})`     | The bus is invoking the handler locally.                                                                                                                                       |
+| `${RepositoryClass}.load ${identifier}`         | The aggregate is being loaded by the repository. During this time Axon Framework will obtain a lock, fetch snapshots and events from the event store to hydrate the aggregate. |
+| `LockingRepository.obtainLock`                  | The repository is obtaining a lock for the aggregate. This taking some time indicates that the command was queued due to another command being handled for the same aggregate. |
+
+During handling of commands, other functionality might be invoked such as scheduling deadlines or publishing events. 
+Please refer to the specific sections of this functionality for more information.
+
+### Events
+
+Publishing events during command execution happens in two steps:
```

Comment:
Although I know the chance is slim, event publication is not restricted to command handling only.  Users can invoke the `EventGateway` too.
I'd expect this section to provide that scenario, and trace sample, too.